documentation/source/user_section/installation/linux.rst

```diff
@@ -29,270 +29,292 @@ For example on Debian/Ubuntu:
 
 .. code:: sh
 
-  apt install gcc
+    apt install gcc
 
 
 On CentOS/RedHat:
 
 .. code:: sh
 
-  yum -y install gcc
+    yum -y install gcc
 
 
 Installation Options
 ====================
 
+Depending on your use case, there are a number of different installation methods available:
 
-Option 1: Install from Package (recommended)
---------------------------------------------
+- :ref:`Packaged Installation <native-install>`: For general use.
+- :ref:`GitHub Installation <github-install>`: For developers.
+- :ref:`Docker Installation <docker-install>`: For Cross-Platform Use/Testing.
+- **[Experimental]** :ref:`Apptainer Installation <apptainer-install>`: For HPC use and other restricted permission applications.
```

Comment:
Maybe it would be good to also use the word "cluster" here just because it gives the user more keywords to go "aha, this is my use-case!"

```suggestion
- **[Experimental]** :ref:`Apptainer Installation <apptainer-install>`: For HPC clusters and other restricted permission applications.
```